src/logics/Logic.cc

```diff
@@ -1418,10 +1420,10 @@ Logic::dumpFunction(ostream& dump_out, const TemplateFunction& tpl_fun)
     const vec<PTRef>& args = tpl_fun.getArgs();
     for (PTRef arg : args) {
         char* arg_name = printTerm(arg);
-        dump_out << '(' << arg_name << ' ' <<  getSortName(getSortRef(arg)) << ") ";
+        dump_out << '(' << arg_name << ' ' <<  sort_store.printSort(getSortRef(arg)) << ") ";
         free(arg_name);
     }
-    dump_out << ") " << getSortName(tpl_fun.getRetSort());
+    dump_out << ") " << sort_store.printSort(tpl_fun.getRetSort());
```

Comment:
Also here I would call just the `Logic`'s method.
```suggestion
    dump_out << ") " << sort_store.printSort(tpl_fun.getRetSort());
```

Reply:
Fixed.